MessengerProj/src/main/res/values/strings.xml

```diff
@@ -110,27 +110,27 @@
     <!--cache view-->
     <string name="CacheSettings">Storage</string>
     <string name="KeepMedia">Keep Media</string>
-    <string name="KeepMediaInfo">Photos, videos and other files from cloud chats that you have <![CDATA[<b>not accessed</b>]]> during this period will be removed from this device to save disk space.</string>
+    <string name="KeepMediaInfo">Photos, videos and other files from cloud chats you have <![CDATA[<b>yet to access</b>]]> during this period will be removed from this device to save disk space.</string>
     <string name="KeepMediaForever">Forever</string>
     <!--passcode view-->
-    <string name="Passcode">Passcode Lock</string>
-    <string name="ChangePasscode">Change Passcode</string>
-    <string name="ChangePasscodeInfo">When you set up an additional passcode, a lock icon will appear on the chats page. Tap it to lock the app.\n\nNote: if you forget the passcode, you\'ll need to delete and reinstall the app.</string>
+    <string name="Passcode">App Lock</string>
```

Comment:
Yes - it is the app that is locked, not the passcode. Sounds reasonable to me.

Reply:
To me this is the "Lock Passcode"

Reply:
@Ampli-fier How I read that is "to lock the passcode" rather than "a passcode for the lock".

Reply:
It might be my german brain :)

What can I do as a user, when I press this button?

"Set the passcode" 
"Set the passcode to lock the app"

in short:

"Set the lock passcode"

shorter:

"Lock Passcode"

Reply:
@Ampli-fier The problem arises from trying to go back the other way, especially in a language that does not use compound-words correctly.

Reply:
"Set the lock passcode" probably avoids this for most languages (which I cannot _really_ tell only knowing two).